src/products/mozilla_org/testing.md

```diff
@@ -97,28 +131,38 @@ On the right side of the page:
-* Open the browser by selecting either New Window or New Private Window.
-* Copy and paste [step one](https://www.mozilla.org/firefox/68.0.1/tracking-protection/start/?step=1&variation=0) URL to your browse, then hit the “return” or “enter” keyboard button. A text box pops up.
-* Click the **Next** button, which takes you to steps [two](https://www.mozilla.org/firefox/68.0.1/tracking-protection/start/?step=2&variation=1) and [three](https://www.mozilla.org/firefox/68.0.1/tracking-protection/start/?step=3&variation=2).
-* In step [three](https://www.mozilla.org/firefox/68.0.1/tracking-protection/start/?step=3&variation=2), an extra text box appears which shows your current Content Blocking settings of the page.
-* Click on **Got it!** button to finish step three.
-* Click on **Restart tour** button to go through the above steps.
-* Click on the [FAQ](https://support.mozilla.org/kb/what-happened-tracking-protection) link which takes you to the topic on SUMO site.
-
-### [navigation.lang](https://www.mozilla.org/)
+### [navigation.ftl](https://www.mozilla.org/)
 
 This page is activated on production whether it is localized or not. It is not an independent page but a file of shared content. When clicking on any of these topics on the navigation bar, **Firefox**, **Projects**, **Developers** or **About**, you will see a drop-down window with subcategory topics. Within each of the topics, there is a description and a few links to help you dive deeper further into a topic which takes you to a different page. When reviewing the content, keep in mind the following:
 * Not all links take you to a site that’s localizable for all locales.
 * Not all products are offered in your locale.
 * Brand and product names must remain unchanged.
+
+## Fixing issues in pages not migrated to Fluent
+
+Migration of all the pages from .lang to Fluent will take some time to complete. The pages in the old format are disabled on Pontoon. Unless there is a glaring mistake or a serious layout issue in localized content, please refrain from making change requests. If changes are indeed needed, here are the steps to get it fixed. If it is brand or product name related correction, please wait until the page is migrated.
+
+### Fixing it through GitHub
+
+* Search in https://transvision.mozfr.org/ for the string or part of the string to correct.
+* Locate the file name where the string comes from.
+* Find the file in the [mozilla.org legacy repo](https://github.com/mozilla-l10n/www.mozilla.org).
+* Create a PR and invite a peer to review and approve the suggestion
+* Once the PR is approved, the l10n-driver in charge will merge the PR to master.
+
+### Fixing it through Bugzilla
+
+* File a bug in [Bugzilla](https://bugzilla.mozilla.org/enter_bug.cgi?product=Mozilla%20Localizations) and file under your locale.
+* Search in https://transvision.mozfr.org/ for the string or part of the string to correct.
```

Comment:
Technically there is also the Report a bug feature, but it goes in a different project (www.mozilla.org::l10n).